README.md

```diff
@@ -90,7 +90,7 @@ steps:
 - run: pnpm test
 ```
 
-> At the moment, only `lock` files in the project root are supported.
+For more examlpes of caching, please see the [Advanced usage](docs/advanced-usage.md#caching-packages-dependencies) guide.
```

Comment:
I think we should remove this note because currently it looks like it is related to only pnpm

Reply:
Thanks!